source/Threema/MsgApi/PublicKeyStore.php

```diff
@@ -62,4 +63,11 @@
 	 * @return bool
 	 */
 	abstract protected function savePublicKey($threemaId, $publicKey);
+    
+	/**
+	 * Initialize a new File|PhpFile Public Key Store
+	 * @param string $path
+	 * @return File|PhpFile
+	 */
+	abstract protected function create($path);
```

Comment:
As the create method in the File/PHPFile keystores are also public you should also make the abstract method _public_ and also make it _static_ (like the File/PHPFile keystores), otherwise PHP throws an error (`Cannot make non static method Threema\MsgApi\PublicKeyStore::create() static in class Threema\MsgApi\PublicKeyStores\PhpFile in [...]\PublicKeyStores\PhpFile.php on line 19`.